### DISCUSSION TITLE
PC game port support still planned?

### DISCUSSION THREAD

**Discussion by sofakng** [darthcloud/BlueRetro#790] [General]
Just wondering if PC game port is still planned?

Thanks! :)

**Comment by darthcloud** [comment]
Yes maybe in the next year. More likely to be a ISA/PCI card however.

**Reply by konwektor** [reply]
I think it is already done here : [https://github.com/netham45/BlueRetro/commit/0d5eba45e65fd7cbdf1260486a8294c1d177e811](url)
"Add gameport support using custom board" - but no details about it...

**Comment by sofakng** [comment]
That sounds awesome ... Really looking forward to that!

If you are interested in any testing, I would gladly order and build some PCBs to help test it out!

**Comment by konwektor** [comment]
Im already involved in blueretro+ogx360 fork from this guy, almost done HW2 internal for xbox. 
Bad thing is author of code (netham45) ,doesnt answer at all. I tried to contact with him over rededit without success, and i know from other ppl, they also did not recive any answer. without details about hardware could be difficult to bring this fork to live.
here is short video from Him under other nickname : [https://youtu.be/a8n7kOmYA-A?si=FNrONZen5DpEEdWn](url)
 Anyway, thanks for propose - If You get some details from netham45, or got better than I coding  knowledge, for build it - no problem mate, let me know. I am good in electronic, but totaly noob  in programing :-[
